maxim/tests/test_test_runs.py

```
@@ -241,5 +263,597 @@ def error(self, message: str, e: Optional[Exception] = None) -> None:
         if result is not None:
             self.assertListEqual(result.failed_entry_indices, [])
 
+    def test_create_test_run_with_image_variables_and_prompt_workflow(self):
+        """Test prompt workflow and chains with image variable columns and other variable types"""
+
+        # Test data with multiple variable types including image URLs
+        data: Data = [
+            {
+                "input": "Analyze this image and text",
+                "image_url": "https://images.unsplash.com/photo-1494871262121-49703fd34e2b?fm=jpg&q=60&w=3000&ixlib=rb-4.1.0&ixid=M3wxMjA3fDB8MHxzZWFyY2h8NHx8dGV4dHxlbnwwfHwwfHx8MA%3D%3D",
```

Pylint (3.3.7)

[convention] Line too long (188/100)

(C0301)